projects/online/online/main.py

```diff
@@ -387,8 +394,11 @@ def main(
             Input data sample rate in Hz
         kernel_length:
             Length of Aframe analysis kernel in seconds
-        inference_sampling_rate:
+        online_inference_rate:
```

Comment:
Can you add to this saying something like "determines the timing resolution with which Aframe can analyze events"